features/basic_replication.feature

```diff
@@ -77,7 +77,7 @@ Feature: basic replication
     Then I receive a response returncode 0
     And postgres2 role is the primary after 24 seconds
     And Response on GET http://127.0.0.1:8010/history contains recovery after 10 seconds
-    When I issue a PATCH request to http://127.0.0.1:8010/config with {"synchronous_mode": null, "master_start_timeout": 0}
+    When I issue a PATCH request to http://127.0.0.1:8010/config with {"synchronous_mode": null, "master_start_timeout": 0, "failsafe_mode": true}
     Then I receive a response code 200
```

Comment:
Why do we only rely on the response code, isn't making sure the configuration param has applied successfully (by issuing GET /config request) necessary?

Reply:
Because we rely on the implementation of RestApiHandler.do_PATCH_config(). If something is wrong the HTTP status code is different from 200.